recipes-core/images/iot2050-image-example.bb

```diff
@@ -108,3 +108,11 @@ IMAGE_INSTALL += " \
     node-red-gpio \
     node-red-preinstalled-nodes \
     "
+
+IOT2050_CORAL_SUPPORT ?= "1"
+
+IMAGE_INSTALL += "${@ ' \
+    python3-pycoral \
```

Comment:
do we know the version of these package?  seems there is no snapshot for the coral package.
could we lock the package version?

Reply:
The locking is done via meta-coral, which we lock via kas. If you see an issue in version locking of meta-coral, please report there first.